### DISCUSSION TITLE
After satellite change detection, how can I score the extent of the change?

### DISCUSSION THREAD

**Discussion by sunhoro** [satellite-image-deep-learning/techniques#37] [General]
Hello all,

Is there a way to roughly score the extent of the change?
I need to somehow filter out the dataset that showed minimal change.

For example, I have a bunch of data sets for new construction detection.
Then after processing out the change detection, how can I automatically filter the locations that had new constructions?

**Comment by robmarkcole** [comment]
Typically the pixels that have changed are highlighted, so you can simply count them. The challenge is usually to identify meaningful change, e.g. floodwater, new construction.

**Reply by sunhoro** [reply]
I'm thinking of a process like removing small pixels, then computing white&black ratio.....
@robmarkcole do you know any simple ways to calculate black and white pixel ratio of an image?

**Comment by johnnqasim** [comment]
Despite their significance in the online gambling landscape, Toto sites are not without their challenges. The constant evolution of technology and the emergence of new gambling platforms [메이저놀이터 ](http://juntt2024.com) pose ongoing challenges for Toto site operators in maintaining up-to-date verification standards. Additionally, the proliferation of illegal gambling sites requires continuous efforts to combat illicit activities and protect users from potential harm.
